web/src/features/hostdashboard/hostpricing/components/PriceInput.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const PriceInput = ({ 
+  value, 
+  onChange, 
+  disabled = false, 
+  ariaLabel = "Price input",
+  className = "",
```

Comment:
Waarom is deze classname leeg?
```suggestion
  className = "",
```

Reply:
Heb dit aangepast/eruitgezet